cmd/env.go

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2016 Tom Whiston <tom.whiston@gmail.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+// envCmd represents the env command
+var envCmd = &cobra.Command{
+	Use:   "env",
+	Short: "add or remove env variables",
+	Long: `Easily add or remove environement variables
+
+	This allows you to easily configure Blackfire environment`,
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
you can remove this actually. If you completely remove Run from the struct it will just show the help anyway. See config.go for an example